packages/multicast_dns/lib/src/resource_record.dart

```diff
@@ -162,25 +145,22 @@ class ResourceRecordQuery {
   bool get isUnicast => questionType == QuestionType.unicast;
 
   /// Encodes this query to the raw wire format.
-  List<int> encode() {
-    return encodeMDnsQuery(
-      fullyQualifiedName,
-      type: resourceRecordType,
-      multicast: isMulticast,
-    );
-  }
+  List<int> encode() => encodeMDnsQuery(
```

Comment:
This change doesn't match the style guide. Here and below

Reply:
Got it, reverted.